lib/tasks/sync_measurements_counter.rake

```diff
@@ -0,0 +1,6 @@
+desc "Sync measurements count on streams"
```

Comment:
fix.rake is not a good name but I guess it could be a good place to keep all this "fix"ing tasks.